src/nedap/utils/test/matchers.cljc

```diff
@@ -0,0 +1,10 @@
+(ns nedap.utils.test.matchers
```

Comment:
As it is right now, `utils.test` consumers would need to depend on `matcher-combinators` manually

That seems perfect to me, since m-c performs a side-effect into the global environment; that should be opt-in.

Seems worth a think and also reflecting in the readme

Reply:
That's the idea; i'm just curious how to prevent the `lein check`-step in the CI from failing. Obviously the m-c is not available. Could add `+test` in that case.

WDYT?